IZ and CWS
o http://www.openoffice.org/issues/show_bug.cgi?id=41006
o CWS (not yet)

offapi/drafts/com/sun/star/rendering/XParametricPolyPolygon2DFactory.idl 
contains an illigal char

Index: offapi/drafts/com/sun/star/rendering/XParametricPolyPolygon2DFactory.idl
===================================================================
RCS file: /cvs/api/offapi/drafts/com/sun/star/rendering/XParametricPolyPolygon2DFactory.idl,v
retrieving revision 1.4
diff -u -r1.4 XParametricPolyPolygon2DFactory.idl
--- offapi/drafts/com/sun/star/rendering/XParametricPolyPolygon2DFactory.idl	13 Jan 2005 18:04:20 -0000	1.4
+++ offapi/drafts/com/sun/star/rendering/XParametricPolyPolygon2DFactory.idl	19 Jan 2005 23:45:31 -0000
@@ -109,7 +109,7 @@
 		linearly between center and endColor via concentric ellipses
 		around the center point. Note that the absolute size of the
 		rectangle given at this method does not matter, only it's
-		aspect ratio: if this aspect ratio ís 1, the gradient is
+		aspect ratio: if this aspect ratio is 1, the gradient is
 		circular. If it's not 1, the gradient is elliptical, with the
 		special twist that the aspect ratio is maintained also for the
 		center color: the gradient will not collapse into a single
@@ -124,7 +124,7 @@
 		linearly between center and endColor via rectangular boxes
 		around the center point. Note that the absolute size of the
 		rectangle given at this method does not matter, only it's
-		aspect ratio: if this aspect ratio ís 1, the gradient is
+		aspect ratio: if this aspect ratio is 1, the gradient is
 		quadratic. If it's not 1, the gradient is rectangular, with
 		the special twist that the aspect ratio is maintained also for
 		the center color: the gradient will not collapse into a single
